# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/libsdl_sound/hotfix-physfs3.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

PhysicsFS 3 no longer defines __EXPORT__, the playsound helpers are local anyway.

diff -Nrup SDL_sound-1.0.3.orig/playsound/physfsrwops.h SDL_sound-1.0.3/playsound/physfsrwops.h
--- SDL_sound-1.0.3.orig/playsound/physfsrwops.h	2008-04-17 19:56:21.000000000 +0200
+++ SDL_sound-1.0.3/playsound/physfsrwops.h	2026-09-27 18:13:46.773342625 +0200
@@ -39,7 +39,7 @@ extern "C" {
  *  @return A valid SDL_RWops structure on success, NULL on error. Specifics
  *           of the error can be gleaned from PHYSFS_getLastError().
  */
-__EXPORT__ SDL_RWops *PHYSFSRWOPS_openRead(const char *fname);
+SDL_RWops *PHYSFSRWOPS_openRead(const char *fname);
 
 /**
  * Open a platform-independent filename for writing, and make it accessible
@@ -51,7 +51,7 @@ __EXPORT__ SDL_RWops *PHYSFSRWOPS_openRe
  *  @return A valid SDL_RWops structure on success, NULL on error. Specifics
  *           of the error can be gleaned from PHYSFS_getLastError().
  */
-__EXPORT__ SDL_RWops *PHYSFSRWOPS_openWrite(const char *fname);
+SDL_RWops *PHYSFSRWOPS_openWrite(const char *fname);
 
 /**
  * Open a platform-independent filename for appending, and make it accessible
@@ -63,7 +63,7 @@ __EXPORT__ SDL_RWops *PHYSFSRWOPS_openWr
  *  @return A valid SDL_RWops structure on success, NULL on error. Specifics
  *           of the error can be gleaned from PHYSFS_getLastError().
  */
-__EXPORT__ SDL_RWops *PHYSFSRWOPS_openAppend(const char *fname);
+SDL_RWops *PHYSFSRWOPS_openAppend(const char *fname);
 
 /**
  * Make a SDL_RWops from an existing PhysicsFS file handle. You should
@@ -75,7 +75,7 @@ __EXPORT__ SDL_RWops *PHYSFSRWOPS_openAp
  *  @return A valid SDL_RWops structure on success, NULL on error. Specifics
  *           of the error can be gleaned from PHYSFS_getLastError().
  */
-__EXPORT__ SDL_RWops *PHYSFSRWOPS_makeRWops(PHYSFS_file *handle);
+SDL_RWops *PHYSFSRWOPS_makeRWops(PHYSFS_file *handle);
 
 #ifdef __cplusplus
 }
